src/springtime/datasets/phenocam.py

```diff
@@ -181,12 +212,25 @@ def _to_geopandas(df):
     sites = list_sites()
     site_locations = sites[["site", "geometry"]]
     df = df.merge(site_locations, on="site")
-    return geopandas.GeoDataFrame(df, geometry=df.geometry)
+    df = geopandas.GeoDataFrame(df, geometry=df.geometry)
+    df.rename(columns={"date": "datetime"}, inplace=True)
+    # Do not return variables that are not derived from image data
+    # to get raw file see phenocam_data_dir directory.
+    non_derived_variables = {
+        # Columns from https://phenocam.nau.edu/data/archive/harvard/ROI/harvard_DB_1000_3day.csv
+        'date','year','doy','image_count','midday_filename',
+        # Columns added by phenocamr
+        'site','veg_type',
```

Comment:
I am not sure about dropping `veg_type`. Looking at columns, how can be derived later.
```
In [2]: df.columns
Out[2]: 
Index(['datetime', 'geometry', 'midday_r', 'midday_g', 'midday_b',
       'midday_gcc', 'midday_rcc', 'r_mean', 'r_std', 'g_mean', 'g_std',
       'b_mean', 'b_std', 'gcc_mean', 'gcc_std', 'gcc_50', 'gcc_75', 'gcc_90',
       'rcc_mean', 'rcc_std', 'rcc_50', 'rcc_75', 'rcc_90', 'max_solar_elev',
       'snow_flag', 'outlierflag_gcc_mean', 'outlierflag_gcc_50',
       'outlierflag_gcc_75', 'outlierflag_gcc_90', 'smooth_gcc_mean',
       'smooth_gcc_50', 'smooth_gcc_75', 'smooth_gcc_90', 'smooth_rcc_mean',
       'smooth_rcc_50', 'smooth_rcc_75', 'smooth_rcc_90', 'smooth_ci_gcc_mean',
       'smooth_ci_gcc_50', 'smooth_ci_gcc_75', 'smooth_ci_gcc_90',
       'smooth_ci_rcc_mean', 'smooth_ci_rcc_50', 'smooth_ci_rcc_75',
       'smooth_ci_rcc_90', 'int_flag'],
      dtype='object')
```

Reply:
You are right, added it back